source/_docs/z-wave/adding.markdown

```diff
@@ -3,6 +3,10 @@ title: "Z-Wave Devices - Adding and Removing"
 description: "How to add and remove Z-Wave devices."
 ---
 
+**Recommendation before adding any devices:**
+
+Z-Wave devices behave as a mesh and can store network relationship details on the device itself.  This means used devices or even brand new devices could already be enrolled in another network, for example a test network for a brand new device or a previous network for devices aren't new. This could cause headaches when you're attempting to add/enrol the device to your network. It is recommended that if possible, perform a factory reset AND device perform exclusion or disenroll for the device you're attempting to add to Home Assistant. Steps can found further below on this page under "Removing Devices".
```

Comment:
```suggestion
Z-Wave devices behave as a mesh and can store network relationship details on the device itself. This means used devices or even brand new devices could already be enrolled in another network, for example, a test network for a brand new device or a previous network for devices isn't new. This could cause headaches when you're attempting to add/enroll the device to your network. It is recommended that if possible, perform a factory reset AND device perform exclusion or disenroll for the device you're attempting to add to Home Assistant. Steps can found further below on this page under "Removing Devices".
```

Reply:
At a glance this looks like no changes were recommended? I'll resolve it for now but let me know if I should make changes, will be happy to.

Reply:
there is a `,` comma missing after `, for example`, that is what this suggestion is adding.

Reply:
got it thanks! also updating a section that I said "Previous devices aren't new" that should be "previous devices that aren't new" pushing this commit